apps/aecore/src/aec_chain.erl

```diff
@@ -120,8 +120,9 @@ stop() ->
 %% TODO: handle out-of-order blocks
 -spec top() -> {ok, block()}.
 top() ->
-    {ok, {Height, _Trees}} = aec_state:get_trees(),
-    {ok, _BlockWithState} = get_block_by_height(Height).
+    {ok, {Height, Trees}} = aec_state:get_trees(),
+    {ok, BlockWithoutState} = get_block_by_height(Height),
```

Comment:
Well spotted!

Reply:
@lucafavatella I keep forgetting that we cache blocks in network format. We need to prioritize work on that